docs/developer/README.md

```
@@ -47,6 +47,23 @@ To deploy with IdP provided organizations configured:
+```
+
+To deploy with auth enabled (uses OIDC with PAM issuer):
+```
+AUTH=true make deploy-quadlets
+```
+
+To deploy with organizations support enabled:
+```
+AUTH=true ORGS=true make deploy-quadlets
+```
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)